source/_integrations/nest.markdown

```diff
@@ -297,7 +297,7 @@ In this section you will authorize Home Assistant to access your account by gene
 
 - Restart the Thermostat device. See [How to restart or reset a Nest thermostat](https://support.google.com/googlenest/answer/9247296) for more details.
 - In the official Nest app or on https://home.nest.com: Move the Thermostat to a different or fake/temporary room.
-- Reload the integration in Home Assistant:  Navigate to **Configuration** then **Integrations**, click `...` next to *Nest* and choose **Reload**.
+- Reload the integration in Home Assistant:  Navigate to **Configuration** then **Devices & Services**, click `...` next to *Nest* and choose **Reload**.
```

Comment:
Looks good for nest docs. Looking above we also have a reference to an icon that doesn't exist anymore below they My icon explaining the steps to get to integrations to add it so maybe consider this is another pass:


```
1. In the sidebar click on _**{% my config icon %}**_.
1. From the configuration menu select: _**{% my integrations icon %}**_.
```